kfc/kfc-gradle-plugin/src/main/kotlin/com/github/turansky/kfc/gradle/plugin/PatchWebpackConfig.kt

```diff
@@ -75,21 +79,32 @@ open class PatchWebpackConfig : DefaultTask() {
         )
     }
 
+    fun replace(
+        oldValue: String,
+        newValue: String,
+        strict: Boolean = false,
```

Comment:
Redundant?

Reply:
We don't use it, but of all the options of string-replace-loader, `strict` seemed like a reasonable choice to support from the beginning.

Reply:
I would prefer to see `strict` support in separate (next) PR

Reply:
Fine by me, I will remove it for now.

Reply:
Removed.